examples/hackers_delight/variants/p2/p2-dead.wat

```diff
@@ -0,0 +1,44 @@
+;; Test whether an unsigned integer is of the form 2^(n-1)
```

Comment:
@anirudhSK 

Here the orignal program has dead code, getting local 0 and setting local 1 with that value, which doesn't affect the return value. 

Some of the equivalent programs also have dead codes and one has nops instead.